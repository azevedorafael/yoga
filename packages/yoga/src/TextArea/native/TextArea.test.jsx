import React from 'react';
import { render } from '@testing-library/react-native';

import { ThemeProvider, TextArea } from '../..';

describe('<TextArea />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ThemeProvider>
        <TextArea />
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
