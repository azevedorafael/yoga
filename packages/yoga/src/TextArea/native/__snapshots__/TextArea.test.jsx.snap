// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextArea /> should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    full={false}
    style={
      Array [
        Object {
          "position": "relative",
          "width": 320,
        },
        Object {},
      ]
    }
  >
    <TextInput
      allowFontScaling={true}
      cleanable={false}
      disabled={false}
      editable={true}
      focus={false}
      full={false}
      multiline={true}
      rejectResponderTermination={true}
      style={
        Array [
          Object {
            "borderColor": "#D3D3E2",
            "borderRadius": 8,
            "borderStyle": "solid",
            "borderWidth": 1,
            "color": "#9898A7",
            "fontSize": 14,
            "fontWeight": "400",
            "paddingBottom": 16,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 16,
            "width": 320,
          },
          Object {
            "height": 88,
          },
        ]
      }
      textAlignVertical="top"
      typed={false}
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
</View>
`;
